hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -835,12 +835,16 @@ def status(format, type)
           r_source_string = '       SOURCE:'
           r_load_sink = sl.getReplicationLoadSink
           next if r_load_sink.nil?
+          if (r_load_sink.getTimestampsOfLastAppliedOp() == r_load_sink.getTimeStampStarted())
```

Comment:
nit: Can drop the wrapping parens

Reply:
A comment here would also be very appreciated (e.g "If we have applied no operations since we've started replication, assume that we're not acting as a sink and don't print the normal information")